Examples/NewModuleTemplate/plasterManifest.xml

```diff
@@ -53,6 +53,10 @@
         </parameter>
     </parameters>
     <content>
+        <message>
+Scaffold a PowerShell Module with the files required to run Pester tests, build with PSake and publish to the PSGallery.
+
```

Comment:
In this scenario, the only whitespace we remove in the current impl is shown below mostly with `_`.
```xml
<message>____<crlf>
    Here's my first line.`r`n
    Here's my second line with two newlines following`r`n`r`n
    Here's my final line.
<crlf>____</message>
```
So the CRLFs on the first and second line aren't needed.  Well, you want an extra blank line between lines two and three you could specify it like this:
```xml
<message>
    Here's my first line.
    Here's my second line with two newlines following.`n
    Here's my final line.
</message>
```

Or like this:

```xml
<message>
    Here's my first line.
    Here's my second line with two newlines following

    Here's my final line.
</message>
```